util/dashboard/dashboard.py

```diff
@@ -397,13 +397,13 @@ def run():
 
         with open(dashboard_yml, 'w+') as f:
             yaml.dump(data_yml, f)
-        gateway.detach(robot_gateway)
     except Exception:
         logging.exception(f"Creating  dashboard for {ontology_file} failed")
-    try:
-        gateway.close()
-    except Exception:
-        logging.exception("Closing JavaGateway failed")
+    finally:
+        try:
+            gateway.shutdown()
```

Comment:
assuming I'm looking at the right docs page, you may want to pass a `True` arg here if you want to be logging exceptions below (presumably `shutdown()` suppresses them otherwise)

Reply:
Indeed. Thanks for checking it!